packages/twig-to-php-parser/lib/twig-to-php-parser.php

```diff
@@ -71,6 +71,10 @@ function twig_to_php_parser( $patterns_dir_path, $template_dir_path ) {
 		// https://regex101.com/r/ns5kBR/2
 		$include_regex = '/({%\sinclude ")(.*[c-|o-|l-].*\/)(.*)(.twig)(" with )(\w*)(.*\s%})/';
 
+		// Get matches for {% include '/path/here/' ~ variable_svg ~ '.svg' %}
+		// https://regex101.com/r/pdleQb/3
+		$svg_regex = '/({%\sinclude\s.*)(\s?\~\s?)(\S*)(\s?\~\s?)(.*\s.*)(%})/';
```

Comment:
Did it need a seperate regex? Was there any issue using the $include_regex?

Reply:
Hmm, I thought so when I wrote it - the statement structure is different since we aren't passing any data into the SVG, and the include regex looks for a `.twig` a `with` which are not in the SVG include.

That said, looking at it now, there is probably a way to update the include regex to accommodate both cases. Is that worth doing? @asannad

Reply:
No thats fine, as long as we know the reason we are alrite. Approving now.